sentry-core/src/lib.rs

```diff
@@ -14,6 +14,68 @@
 //! the concepts of [`Client`], [`Hub`] and [`Scope`], as well as the extension
 //! points via the [`Integration`], [`Transport`] and [`TransportFactory`] traits.
 //!
+//! # Parallelism, Concurrency and Async
+//!
+//! The main concurrency primitive is the [`Hub`]. In general, all concurrent
+//! code, no matter if multithreaded parallelism, or futures concurrency needs
+//! to run with its own copy of a Hub. Even though the [`Hub`] is internally
```

Comment:
This is the only "Hub" that isn't hyperlinked.
```suggestion
//! to run with its own copy of a [`Hub`]. Even though the [`Hub`] is internally
```